packages/esm-patient-banner-app/src/banner-tags/deceased-tag.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import DeceasedBannerTag from './deceased-tag.component';
+import { mockDeceasedPatient } from '../../../../__mocks__/mockDeceasedPatient';
+
+describe('deceasedTag', () => {
+  it('renders a deceased tag in the patient banner for patients who died', () => {
+    renderDeceasedBannerTag();
+  });
```

Comment:
This is not a test; it does not test anything. Please make your test test something (such as that some text appears).

You should also add a test verifying that it does not appear for patients who are not dead.

Reply:
ok. thank you for making it clear. let me make the changes @brandones